media/css/mozorg/home/fx-quantum-promo.scss

```diff
@@ -0,0 +1,63 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+.quantum-promo {
+    background-color: #45caf3;
+    @include at2x('/media/img/mozorg/promo/quantum-promo-banner-phone.png', 760px, 200px);
+    background-repeat: no-repeat;
+    @include background-size(760px 200px);
```

Comment:
> background-size is already part of the at2x mixin

Indeed but, I have found on more than one occasion that in IE it is not added and then things go all weird with regards to sizing. See here for example where Alex had to add it[1]. Perhaps it is the LESS version?

I still cannot spin up a VM with VirtualBox so, I will remove it here, and if you find it works fine in IE then all is good.

[1] https://github.com/mozilla/bedrock/pull/5141/files#diff-39020cb2b45254eb12cb23557eeb073dL498

Reply:
This only happens if the original size for the assets are larger than the display size. Like Craig says, the mixin should already include background-size either way.

As for removing the mixin, we technically could. Although there are probably millions of old phones out there that can’t be updated to the latest browser versions. If this was something that degraded gracefully it would be a different story, but it results in some pretty broken pages imo.

Reply:
Ah, in that case, the change should be 100% Thanks for clearing that up @alexgibson